clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/config/CloudDriverConfig.groovy

```diff
@@ -81,6 +119,17 @@ class CloudDriverConfig {
     Clock.systemDefaultZone()
   }
 
+  /**
+   * rz - This is a consolidation of mutations made to the default spring objectmapper.
+   * TODO(rz): Remove Autowiring of ObjectMappers from the project entirely.
+   */
+  @Autowired
+  ObjectMapper objectMapper(ObjectMapper objectMapper) {
```

Comment:
was just looking at this:

https://docs.spring.io/spring-boot/docs/current/reference/html/howto-spring-mvc.html#howto-customize-the-jackson-objectmapper

In theory you could replace this bit with some properties in the default environment

fail_on_unknown_properties false should actually be a default

setting
```
spring.jackson.default-property-inclusion=non_null
spring.jackson.serialization.fail_on_empty_beans=false
```

should accomplish the same thing

not sure if that feels like too much magic or not, and the ability to get at this programmatically for tests is also nice so theres that too..

Reply:
if nothing else then maybe just defining a `Jackson2ObjectMapperBuilderCustomizer` Bean here instead of autowiring the objectMapper would be the more booty way to do this

Reply:
Thanks this is neat-o. TIL.